Cargo.toml

```diff
@@ -45,6 +44,7 @@ callee_1 = { path = "tests/contracts/callee-1" }
 callee_2 = { path = "tests/contracts/callee-2" }
 gas_consumed = { path = "tests/contracts/gas_consumed" }
 counter_float = { path = "tests/contracts/counter_float" }
+persistance = { path = "tests/contracts/persistance" }
```

Comment:
Is there really a need for a separate contract?
The persistence is part of the network state, does not involve any change in the contract. You can use any contract you want to test it – and more than one – that already exists.

Also: we cannot fix the typo in microkelvin atm without a new release, but at least we can avoid to spread the typo in other places, so whenever possible let's use the correct word "persistence".

Reply:
kk I'll just use the counter then